packages/legend-extension-dsl-mastery/src/graph-manager/protocol/pure/v1/transformation/pureGraph/from/V1_DSL_Mastery_TransformerHelper.ts

```diff
@@ -819,3 +821,28 @@ export const V1_transformMasterRecordDefinition = (
     element.exceptionWorkflowTransformService;
   return protocol;
 };
+
+/**********
+ * runtime
+ **********/
+
+export const V1_transformMasteryRuntime = (
+  element: MasteryRuntime,
+  context: V1_GraphTransformerContext,
+): V1_MasteryRuntime => {
+  const extraMasteryRuntimeTransformers = context.plugins.flatMap(
+    (plugin) =>
+      (
+        plugin as DSL_Mastery_PureProtocolProcessorPlugin_Extension
+      ).V1_getExtraMasteryRuntimeTransformers?.() ?? [],
+  );
+  for (const transformer of extraMasteryRuntimeTransformers) {
+    const protocol = transformer(element, context);
+    if (protocol) {
+      return protocol;
+    }
+  }
+  throw new UnsupportedOperationError(
+    `Can't transform runtime '${typeof element}'`,
```

Comment:
Please modify error message to: `Can't transform mastery runtime: : no compatible transformer available from plugins, element`